### DISCUSSION TITLE
Lvgl Display over MIPI DBI SPI with DMA

### DISCUSSION THREAD

**Discussion by iamadahn** [zephyrproject-rtos/zephyr#81873] [Q&A] [answered]
I'm trying to use display with lvgl over MIPI DBI SPI and everything works correctly, but the problem is that the transmission is quite slow, and the more pixels there are to update the slower it updates. The solution is obvious - I have to use DMA, but I wasn't able to find how to configure MIPI DBI SPI with DMA. I'm using stm32h723vg soc with dmamux, and when I enable DMA, SPI just stops working at all, no sign of life at all according to oscilloscope's screen. So I'm asking if it is possible at all, and if someone was able to succeed, please share your experience

**Comment by cempod** [comment]
Hello,

I've been trying to solve that issue too, but no luck yet. Hope that we will get an answer here.

**Comment by Kirix1004** [comment]
I am currently working with the same task, by the way I have st7796 display controller and I'm struggling with dma too, any help will be greatly appreciated!

**Comment by JarmouniA** [comment, marked as answer]
@iamadahn @Kirix1004 see #82095
